app/controllers/events_controller.rb

```diff
@@ -204,4 +237,46 @@ def rss
       format.rss { render layout: false }
     end
   end
+
+  private
+
+  def display_all_parameter_options(events)
```

Comment:
If you use Event::EVENT_TYPE and Area.all i dont see think this method will be used